Tech Logs for Flow 1 and Flow 2 for TRV11

Nirdosh3312 opened the pull request:
Submitting Tech logs for Flow 1 and Flow 2 for ONDC:TRV11 for verification

Nirdosh3312 commented:
Hi @sourabhsaini11, I have fixed the Issue that you have mentioned. Kindly verify the updated logs

sourabhsaini11 commented:
@Nirdosh3312, please get the below issue fixed
confirm
- additional tag-group 'SETTLEMENT_DETAILS' in payments

KapilKharera commented:
Submitting Tech logs for Flow 1 and Flow 2 for ONDC:TRV11 (METRO) for verification

sourabhsaini11 commented:
@Nirdosh3312, please get the below issues fixed
common_issue
- selected.count type should be int instead of string at quote.breakup

search(fare)
- code 'BAHADURGARH CITY' & 'BADKAL MOR' doens't match with the code sent in past call, at fulfillment.stops

select
- selected.count type should be int instead of string at items

on_select
- selected.count type should be int instead of string at items

init
- selected.count type should be int instead of string at items

on_init
- selected.count type should be int instead of string at items

confirm
- selected.count type should be int instead of string at items
- tags SETTLEMENT_TYPE, DELAY_INTEREST are missing in SETTLEMENT_TERMS tag-group

on_confirm
- tags SETTLEMENT_TYPE, DELAY_INTEREST are missing in SETTLEMENT_TERMS tag-group
- selected.count type should be int instead of string at items

Nirdosh3312 commented:
Hii @sourabhsaini11, I have updated the logs and fixed the issues that you have mentioned. Kindly review the logs and PR. Looking forward to your response.

sourabhsaini11 commented:
@Nirdosh3312, please get the below issues fixed
search(fare)
- code value should refer to station code(MNGM) as sent in previous call instead of station name(MOHAN NAGAR), at location.descriptor

on_init
- tag SETTLEMENT_TYPE is missing in SETTLEMENT_TERMS tag-group

on_cancel
- tag SETTLEMENT_TYPE is missing in SETTLEMENT_TERMS tag-group

Nirdosh3312 commented:
Hii @sourabhsaini11, 

Please help us verify this 

> search(fare)
> - code value should refer to station code(MNGM) as sent in previous call instead of station name(MOHAN NAGAR), at location.descriptor


 As per the swagger example, In search (fare) the stations are given as names only (attaching image for reference). Do we need to change them to codes ?

![Screenshot 2024-05-27 161308](https://github.com/ONDC-Official/verification-logs/assets/150114301/c5bd826c-3148-4d6a-819c-4de9d9356f2d)

> on_init
> - tag SETTLEMENT_TYPE is missing in SETTLEMENT_TERMS tag-group

> on_cancel
> - tag SETTLEMENT_TYPE is missing in SETTLEMENT_TERMS tag-group

- In Swagger SETTLEMENT_TYPE is part of SETTLEMENT_DETAILS, hence we are sending it separately rather than in SETTLEMENT_TERMS, do we need to send it in both the objects ?
